src/frontend/komponenter/Fagsak/TilkjentYtelse/TilkjentYtelse.tsx

```diff
@@ -56,10 +57,16 @@ const TilkjentYtelse: React.FunctionComponent<ITilkjentYtelseProps> = ({
     const { erLesevisning } = useBehandling();
     const { settFagsak } = useFagsakRessurser();
     const { toggles } = useApp();
+    const { utledBehandlingresultat } = useFagsakApi(
+        _ => {
+            'Feilmelding';
+        },
+        _ => {
+            'Feilmelding';
```

Comment:
Burde jeg flytte utledBehandlingresultat-funksjonen til en annen fil enn useFagsakApi for å unngå å putte inn disse rare funksjonene som useFagsakApi må ta inn?